Packs/Jira/Integrations/JiraEventCollector/JiraEventCollector.py

```diff
@@ -152,8 +152,9 @@ def set_next_run(log: dict) -> dict:
         last_run = demisto.getLastRun()
 
         if not last_run.get('next_time'):
-            last_time = log.get('created', '').removesuffix('+0000')
-            next_time = last_time[:-1] + str(int(last_time[-1]) + 1)
+            last_datetime = log.get('created', '').removesuffix('+0000')
+            last_datetime_with_delta = datetime.strptime(last_datetime, DATETIME_FORMAT) + timedelta(milliseconds=1)
+            next_time = datetime.strftime(last_datetime_with_delta, DATETIME_FORMAT)
 
```

Comment:
I changed it to avoid duplicates in case the time is `00:00:00.009` in the old way the increase time will be `00:00:00.0010` which is actually lower. While in the new way it will be `00:00:00.010` which is greater.